web/mocklogin/vite-plugin.ts

```diff
@@ -69,6 +69,8 @@ function mintToken({ sub, party }: TokenRequest): string {
       applicationId: "damlhub",
       actAs: [party],
     };
+  } else {
+    payload.aud = `https://daml.com/jwt/aud/participant/sandbox`;
```

Comment:
why sandbox?  this is local case fall through?

Reply:
yup… this code is _definitely_ not responsible for minting tokens in "prod"

Reply:
I learned the hard way that HTTP JSON API actually cares that an `aud` claim of some kind is required… otherwise the parsing logic fails to detect it as the right kind of token and I get 401's, even against the un-authed local sandbox.